presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -476,6 +478,23 @@ private void printWindowOperatorStats(int indent, WindowOperatorStats stats)
         output.append('\n');
     }
 
+    private static String formatDoubleAsCpuCost(double value)
```

Comment:
I think we can simply rename these methods as `formatCpuCost` and `formatDataSize`, because the parameters are `double` so we don't need to repeat that we are formatting double values.